pkg/fs/mount_linux.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package fs
+
+import (
+	"strings"
+	"syscall"
+)
+
+func (f mountFlags) String() string {
```

Comment:
Nit, doc comment. Suggest `String returns a human readable representation of mountFlags based on which bits are set. E.g. for a value of  syscall.MS_RDONLY|syscall.MS_BIND it will print "MS_RDONLY|MS_BIND"`